recipes-bsp/barebox/barebox-hosttools.inc

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2020 Roland Hieber, Pengutronix <rhi@pengutronix.de>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+SUMMARY = "barebox host tools - imx-usb-loader, omap-usb-loader, bareboximd, and the like"
+HOMEPAGE = "https://www.barebox.org/"
+SECTION = "bootloaders"
+
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=f5125d13e000b9ca1f0d3364286c4192"
+
+SRC_URI = "http://barebox.org/download/barebox-${PV}.tar.bz2"
+S = "${WORKDIR}/barebox-${PV}"
+
+BBCLASSEXTEND = "native nativesdk"
+
+# note: openssl is only needed to build mxsimage, which is not installed
```

Comment:
That's quite unfortunate. Can `mxsimage` be disabled in the KConfig and removed from the defconfig?

Reply:
`mxsimage` is pulled in through `CONFIG_MXS_HOSTTOOLS`, which is enabled to build `mxsboot`… I guess that could be split up.

Reply:
Ah wait. ``mxsboot`` is only needed to build a bootable image for MXS platforms, so it is mostly not needed on host systems out of the barebox tree, and we should not need to install it at all.

Reply:
Should be fixed now.